.gitlab-ci.yml

```diff
@@ -121,7 +121,7 @@ check-pr-DOMjudge:
       - docker/**/*
   script:
     - HUBURL="https://registry.hub.docker.com/v1/repositories/domjudge/domserver/tags"
-    - apk add jq curl
+    - apk add jq curl bash
```

Comment:
if you keep the `sh ./build` I think adding bash is not strictly needed.

Reply:
But then you get a weirdness on Debian / Ubuntu-esque systems: `sh` may not have `[[` available and the build fails there. (The build.sh script would call `sh build-judgehost.sh`, ignoring the shebang there).

I can of course also revert the change to `build.sh`, I just think it might be useful.

Reply:
Makes sense.

Reply:
I've not followed the conversation in detail, but looking at these scripts, is there any reason for making them bash scripts instead of pure posix shell? All the `[[ -f foobar ]]` can simply be replaced by `[ -f foobar ]`.

Reply:
Hmmm, but `set -euxo pipefail` is not POSIX, right? (I know very little about what is POSIX though).

Reply:
`set -eux` does exist in posix, see https://man7.org/linux/man-pages/man1/set.1p.html

Reply:
Ah, I'll adapt accordingly

Reply:
Ok, it seems to work now both locally and on the CI and I think it should all be posix.